README.md

```diff
@@ -267,28 +267,32 @@ spec:
 Create the above pipeline resources via the OpenShift web console or by running the following:
 
 ```bash
-$ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/pipeline/02-resources.yaml
+$ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/pipeline/resources.yaml
 ```
 
 You can see the list of resources created using the CLI:
 
 ```bash
 $ tkn resource ls
 
-NAME              TYPE    DETAILS
-petclinic-git     git     url: https://github.com/spring-projects/spring-petclinic
-petclinic-image   image   url: image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/spring-petclinic
+NAME               TYPE    DETAILS
+voting-api-repo    git     url: http://github.com/sthaha/vote-api.git
```

Comment:
Should we move the app code into the `openshift/openshift-tutorial` repo? or like under this org `https://github.com/openshift-pipelines`?